go/pserver/etcd_client.go

```diff
@@ -64,11 +71,11 @@ func (e *EtcdClient) Register() (int, error) {
 		log.Debugf("inited client to %s", e.etcdEndpoints)
 		break
 	}
-	// init /ps_desired using transaction, for multiple pservers may want to write
+	// init /ps/desired using transaction, for multiple pservers may want to write
```

Comment:
We use `ps_desired` in design doc: https://github.com/PaddlePaddle/Paddle/tree/develop/doc/design/cluster_train#parameter-server-process-1